plugins/algebra/include/detray/array/include/plugins/algebra/array_definitions.hpp

```diff
@@ -8,11 +10,14 @@ namespace detray {
     using scalar = algebra::scalar;
 
     template <typename value_type, unsigned int kDIM>
-    using darray = algebra::array_s<value_type, kDIM>;
+    using darray = std::array<value_type, kDIM>;
 
     template <typename value_type>
-    using dvector = algebra::vector_s<value_type>;
+    using dvector = vecmem::vector<value_type>;
```

Comment:
Is this rebased on #87 ?

Reply:
thanks for pointing this out. Fixed now

Reply:
Looks like we also have to implement vector of vector type in algebra-plugin
(`vecmem: jagged_vector<>` and `std::vector<std::vector<>>`)
I will file a issue for this